RELEASE/scripts/autoscend/quests/level_12.ash

```diff
@@ -1909,29 +1809,7 @@ boolean L12_themtharHills()
 		return autoLuckyAdv($location[The Castle in the Clouds in the Sky (Top Floor)]);
 	}
 
-	buffMaintain($effect[Disco Leer], 10, 1, 1);
```

Comment:
was it intentional to keep acquiring of inhaler here in lvl 12? Guess analogous question - is nuns the only place we want to use inhaler?